tests/Doctrine/Tests/DBAL/Platforms/AbstractPostgreSqlPlatformTestCase.php

```diff
@@ -19,7 +19,7 @@ public function getGenerateTableWithMultiColumnUniqueIndexSql()
     {
         return array(
             'CREATE TABLE test (foo VARCHAR(255) DEFAULT NULL, bar VARCHAR(255) DEFAULT NULL)',
-            'CREATE UNIQUE INDEX UNIQ_D87F7E0C8C73652176FF8CAA ON test (foo, bar)'
+            'ALTER TABLE test ADD CONSTRAINT UNIQ_D87F7E0C8C73652176FF8CAA UNIQUE  (foo, bar)'
```

Comment:
This is actually a breaking change, or will existing schema with unique indexes be correctly handled here?

Reply:
Yes. I didn't thought it out well.

_Backward-Remains-Compatible_:

- All operations involve `<indexes>` and `<index>`.
- `<uniqueConstraints>`: Renaming is ok. `<uniqueConstraint name="old_name_constraints" columns="..." />` to `<uniqueConstraint name="new_name_constraints" />`
- Freshly added  `<uniqueConstraints>`

_Things that will BC-Break_ (although won't break production, will likely break their builds):

- Existing schema that uses `<uniqueConstraints>` generated with `CREATE UNIQUE INDEX indexName ON table (...)` will not be able to be dropped. Those can only be dropped using old syntax `DROP INDEX indexName`, but dbal will generate `ALTER TABLE DROP CONSTRAINT indexName`

This could be totally prevented. Let me work on it.